base/deprecated.jl

```diff
@@ -1699,6 +1699,19 @@ end
 @deprecate_moved varm "StatsBase"
 @deprecate_moved linreg "StatsBase"
 
+# ?? more special functions to SpecialFunctions.jl
+@deprecate_moved gamma "SpecialFunctions"
+@deprecate_moved lgamma "SpecialFunctions"
+@deprecate_moved beta "SpecialFunctions"
+@deprecate_moved lbeta "SpecialFunctions"
+@deprecate_moved lfact "SpecialFunctions"
+function factorial(x::Number)
+    error("""factorial(x::Number) has been moved to the package SpecialFunctions.jl.
```

Comment:
This doesn't make sense to me. Why would we move one method and not the whole function? With this, SpecialFunctions will need to commit type piracy to define that method.

Reply:
Right; https://github.com/JuliaMath/SpecialFunctions.jl/pull/92/files#diff-f839a13f2df5472f7fd840f52ce84e16R757
Do we care though?

Reply:
Seems fine to me. This allows people to do `using SpecialFunctions` in the same REPL session and have it work, which seems far superior to not being able to do so.